psydac/feec/multipatch/operators.py

```diff
@@ -125,8 +133,9 @@ def get_row_col_index(corner1, corner2, interface, axis, V1, V2):
     return row+col
 
 #===============================================================================
-def allocate_matrix(corners, test_space, trial_space):
-
+def allocate_interface_matrix(corners, test_space, trial_space):
+    """ Allocate the interface matrix of a corner domain
+    """
```

Comment:
What is the type of `corners`? What does it contain? What object does it return? As usual, expand the docstring with the usual `Parameters` and `Returns` sections

Reply:
done in [3052219](https://github.com/pyccel/psydac/pull/185/commits/30522190ae19beed5ef084c06b98af4dde21baa7)